cdap-unit-test/src/main/java/co/cask/cdap/test/internal/DefaultStreamManager.java

```diff
@@ -147,6 +157,38 @@ public void send(Map<String, String> headers, ByteBuffer buffer) throws IOExcept
   }
 
   @Override
+  public void send(File file, String contentType) throws Exception {
+    String path = String.format("/v3/namespaces/%s/streams/%s/batch", streamId.getNamespaceId(), streamId.getId());
```

Comment:
StreamClient#sendFile makes HTTP calls, whereas DefaultStreamManager does not start up services, instead makes method calls on handler objects. This is similar to AppFabricClient.deployApplication.